packages/nimble-components/.storybook/preview.js

```diff
@@ -15,12 +15,20 @@ export const parameters = {
     options: {
         storySort: {
             method: 'alphabetical',
+            // Items within arrays show the sort order for children of the category above
+            // https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories
             order: [
                 'Getting Started',
                 'Components',
+                ['Status Table'],
                 'Incubating',
+                ['Docs'],
                 'Tokens',
-                'Tests'
+                ['Docs'],
```

Comment:
tokens doesn't have a child page named Docs, is this just-in-case one is added in the future?

Reply:
Yeah, I figured someone might try to add it and would be confused if it didn't appear at the top but wouldn't know about this mechanism of specifying the order. A lot of what-ifs there, but I think I'll leave it for now.